pages/apply/[resident]/summary.tsx

```diff
@@ -81,31 +81,26 @@ const UserSummary = (): JSX.Element => {
     return <Custom404 />;
   }
 
-  const onConfirmData = () => {
+  const onConfirmData = async () => {
     const [isEligible, reasons] = checkEligible(application);
     if (!isEligible) {
       const reasonStrings = reasons.map((reason) =>
         getDisqualificationReasonOption(reason)
       );
       const reason = reasonStrings.join(',');
 
-      //sends an email, no need to handle save. TODO: add error handling
-      dispatch(sendDisqualifyEmail({ application, reason }));
-
-      setIsSaving(true);
-
-      dispatch(disqualifyApplication(application.id!))
-        .unwrap()
-        .then(() => {
-          router.push('/apply/not-eligible');
-        })
-        .catch(() => {
-          setIsSaving(false);
-          setUserError('Unable to update application');
-          scrollToError();
-        });
+      try {
```

Comment:
Should we add additional tests to ensure `sendDisqualifyEmail` call can fail without cancelling `disqualifyApplication`?